src/ensembl/src/shared/components/upload/Upload.scss

```diff
@@ -21,4 +21,6 @@
   overflow: hidden;
   position: absolute;
   z-index: -1;
+  height: 1px;
+  width: 1px;
```

Comment:
Out of curiosity, will the simple `display: none` do the job? Then there would be no opacity/z-index/height/width nonsense. Judging by [this example on StackOverflow](https://stackoverflow.com/a/29251249/3925302), it should work (we also have an input inside a label).

Reply:
I was following this article here:
https://tympanus.net/codrops/2015/09/15/styling-customizing-file-inputs-smart-way/

It says `the input will be excluded out of tab order` if we use display: none;

Reply:
Good point!

We may probably want to signal that the button is focussed then, because I tried focussing in Storybook, and there was no visual feedback that I am now focussed on the Upload button.

There's a handy `:focus-within` pseudoclass that will target the label that has focussed input inside of it, as in our case. The support is [not perfect](https://developer.mozilla.org/en-US/docs/Web/CSS/:focus-within), but is getting there.